data/map planets.txt

```diff
@@ -3435,6 +3465,17 @@ planet Poisonwood
 		threshold 2000
 		fleet "Large Militia" 7
 
+planet "Pon'tes"
+	government "Hicemus"
+	landscape land/station16
+	"required reputation" 10
+	outfitter "Hicemus"
+	shipyard "Hicemus"
+	description "Pon'tes is uncomfortable, even by the standards of a station. As the Incipias are able to propel themselves through the air, there are no handles or surfaces for you to grip onto. The simulated gravity caused by the rotation of the station is not helping."
+	description "The station is a bit larger than the most human ones and a lot of Incipias are living here. They all seem very busy as they float through the tube-like hallways."
+	spaceport "The station has no normal spaceport - only a small docking area. Most of the people present appear to be workers, as opposed to visiting captains."
+	spaceport "Most of the ships are delivering minerals but some also deliver supplies from Redias. Only a few seem to transport people."
```

Comment:
Didn't they design transport crafts specifically  because of Pon'tes?

Reply:
That does not mean there have to be a lot of them, right?

Reply:
```suggestion
	spaceport "Most of the ships are delivering minerals but some also deliver supplies from Redias. In an small area separated from the rest there is a constant buzzing of small transport ships transporting civilians in both directions."
```